Subdivision_method_3/include/CGAL/Subdivision_method_3/subdivision_masks_3.h

```diff
@@ -331,6 +331,7 @@ class Loop_mask_3 : public PQQ_stencil_3<PolygonMesh, VertexPointMap> {
   void vertex_node(vertex_descriptor vertex, Point& pt) {
     Halfedge_around_vertex_circulator vcir(vertex, *(this->pmesh));
     size_t n = circulator_size(vcir);
+    CGAL_assertion(n > 0);
```

Comment:
Would `vertex != boost::graph_traits<Mesh>::null_vertex()`  also work, which would be more normal?